chore: OPTIC-592:  Migrate any JSX files to include the proper extension

yyassi-heartex opened the pull request:
Updating file extensions to convert `.js` files that have JSX to `.jsx`

+ fixing `create-docs` script to handle `.jsx/.ts/.tsx` as well

### PR fulfills these requirements
- [x] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance (for bug fixes/features)



#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [ ] Backend (Database)
- [ ] Backend (API)
- [x] Frontend

yyassi-heartex committed:
chore: OPTIC-592:  Migrate any JSX files to include the proper extension

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8663866305

yyassi-heartex committed:
clearing out unexpected changes

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8664955808

hlomzik commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/8665095944)
> Successfully merged: Already up to date.

hlomzik committed:
Fix create-docs script to also handle JSX/TS/TSX

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8665718810
